src/components/Member.js

```diff
@@ -0,0 +1,42 @@
+import { useState } from "react";
+
+function Member({
```

Comment:
Throughout this file, you stopped using camelCase in multiple place. Identify all of them throughout this file and the rest of them in your PR and make them all camelCase please

Reply:
As in the first letter of the first word of the function name should be capital too or that `Member` needs to be fixed? I thought it was correct to start the function name with a capital letter that holds all the code?

Reply:
thanks for clarifying!